src/app/components/our-team/our-team.component.html

```diff
@@ -1,24 +1,24 @@
 <app-header-static></app-header-static>
 <div class="container-top">
 	<div class="our-team-container ">
-	<div class="our-team-title">
-		{{coreTeamType}}
-	</div>
+        <div class="our-team-title">
```

Comment:
You can install some extension in  your editor to automatically make proper alignment when you save the file.

Reply:
Yes, and if you use Py-Charm or VSCode, you could easily auto-format.

Reply:
yeah I use VS code